roles/dosomething.phoenix/templates/nginx/app.j2

```diff
@@ -30,6 +30,7 @@ server {
   location / {
     # This is cool because no php is touched for static content
     try_files $uri @rewrite;
+    error_page 404 /leftathetrafficlight.php;
```

Comment:
It's not an actual file. Basically what happens is if an Nginx level 404 is generated, it will send this to the `  location ~ \.php$` directive, which will generate a Drupal 404, since the page doesn't exist as URL alias. I basically made up a name that we'll likely never ever use. I suppose we could use '/404.php', but where's the fun in that?